server/server.go

```diff
@@ -49,11 +47,7 @@ type Server struct {
 // New instantiates a new Server.
 // Callers must then invoke Start to run the Server.
 func New(c *Config) (*Server, error) {
-	out := io.Writer(os.Stdout)
-	if c.Logging.Text {
-		out = zerolog.ConsoleWriter{Out: out}
-	}
-	logger := zerolog.New(out).With().Timestamp().Logger()
+	logger := baseapp.NewLogger(c.Logging)
```

Comment:
Per internal discussion, let's add `Level` to the existing config and shim it here:

```golang
logger := baseapp.NewLogger(baseapp.LoggingConfig{
  Level:  c.Logging.Level,
  Pretty: c.Logging.Text,
})
```